dockerfile-mode.el

```diff
@@ -1,6 +1,8 @@
-;;; dockerfile-mode.el --- Major mode for editing Docker's Dockerfiles
+;;; dockerfile-mode.el --- Major mode for editing Docker's Dockerfiles -*- lexical-binding: t -*-
 
 ;; Copyright (c) 2013 Spotify AB
+;; Package-Requires: ((emacs "24") (s "1.12"))
```

Comment:
Won't this break when emacs gets to version 25?

Reply:
Not at all. These are minimum dependencies, not exact: `package.el` doesn't support the latter.